src/index.js

```diff
@@ -52,6 +54,10 @@ const iSamples_componentPack = {
 	results: {
 		...defaultComponentPack.results,
 		result: iSamplesResult
+	},
+  searchFields: {
+		...defaultComponentPack.searchFields,
+		text: TextSearch
```

Comment:
The indentation looks a bit off in the diff.  In a follow-up PR, we should probably reformat all the source files in the project according to the rules in `.editorconfig`, to make sure things are set up correctly moving forward.  Then, the next time we do a PR we should expect all the diffs to look correct here.

Reply:
That's wired. The index.js shows the correct format if we open it directly but the wrong format in the PR.

Reply:
Yeah, I think there are some hidden tabs somewhere, but we can reformat in a subsequent PR.

Reply:
Get it! Thanks